packages/web-react/src/components/VisuallyHidden/useVisuallyHiddenProps.ts

```diff
@@ -0,0 +1,23 @@
+import { ElementType } from 'react';
+import { useClassNamePrefix } from '../../hooks';
+import { SpiritVisuallyHiddenProps, VisuallyHiddenProps } from '../../types';
+
+export interface VisuallyHiddenStyles<T extends ElementType = 'span'> {
+  /** className props */
+  classProps: string | null;
+  /** props to be passed to the element */
+  props: VisuallyHiddenProps<T>;
+}
+
+export function useVisuallyHiddenProps<T extends ElementType = 'span'>(
+  props: SpiritVisuallyHiddenProps<T>,
+): VisuallyHiddenStyles<T> {
+  const { ...restProps } = props;
+
+  const StackClass = useClassNamePrefix('accessibility-hidden');
```

Comment:
Why `StackClass`? I sense a naming conflict with the `Stack` component here 🤔 I suggest using `visuallHiddenClass` here. The same way as we have it in `Alert` component, for example :-)

Reply:
fixed 😁